package.json

```diff
@@ -1,17 +1,66 @@
 {
   "name": "roboconf-web-administration",
   "version": "0.0.0",
+  "homepage": "http://roboconf.net",
   "description": "Web administration for Roboconf",
-  "main": "index.js",
+  "keywords": [
+    "cloud",
+    "deployment",
+    "orchestration"
+  ],
   "dependencies": {
     "express": "~3.4.7"
   },
   "devDependencies": {
-    "bower": "~1.3.8"
+    "bower": "~1.3.8",
```

Comment:
Actually, I don't use **npm install -p**.
So, I did not have the problem. Production builds are managed by Gulp in my case.

Maybe I don't use these tools correctly...